packages/database/src/realtime/BrowserPollConnection.ts

```diff
@@ -475,7 +477,7 @@ export class FirebaseIFrameScriptHolder {
       const iframeContents = '<html><body>' + script + '</body></html>';
       try {
         this.myIFrame.doc.open();
-        this.myIFrame.doc.write(iframeContents);
+        safeDocument.write(this.myIFrame.doc, sanitizeHtml(iframeContents));
```

Comment:
Again, I would like to verify that this actually works, but I am not sure how to do that beyond running the tests.